src/LiFi.unit.handlers.ts

```diff
@@ -0,0 +1,38 @@
+import { rest } from 'msw'
+import {
+  mockStepTransactionWithTxRequest,
+  mockChainsResponse,
+  mockStatus,
+} from './LiFi.unit.mock'
+import ConfigService from './services/ConfigService'
+import { buildStepObject } from '../test/fixtures'
+
+const config = ConfigService.getInstance().getConfig()
+
+export const lifiHandlers = [
+  rest.post(
+    `${config.apiUrl}/advanced/stepTransaction`,
+    async (_, response, context) =>
+      response(
+        context.status(200),
+        context.json(
+          mockStepTransactionWithTxRequest(
+            buildStepObject({
+              includingExecution: true,
+            })
+          )
+        )
+      )
+  ),
+  rest.get(`${config.apiUrl}/chains`, async (_, response, context) =>
+    response(
+      context.status(200),
+      context.json({
+        chains: mockChainsResponse,
+      })
+    )
+  ),
```

Comment:
Couldn't find where we use this endpoint in tests. 😅

Reply:
This is being called from inside lifi.execute

Reply:
> The api endpoints for this is being called within the lifi.execute method. Hence mocking the response for that over here.

I can't seem to find that place at the first glance. Please share 🙏😄

Reply:
This is being called while creating new instance of `StepExecutionManager`.

Inside the execute method, we are using `ChainService.getInstance()`, which fetches all the chains from the Backend.

Reply:
@Abhikumar98 do we need that huge mock here, or can we just mock something simple just to avoid a call to BE?

Reply:
We'll need Polygon, because that's where the swap is happening. I'll remove the other one